vrk-kpa/suomidigi
# Suomidigi x core 8.8.10 + modules

To test:
- Install the codebase.

Updated quite a bit of modules:
drupal/focal_point (1.4.0 => 1.5.0)
drupal/link_attributes (1.10.0 => 1.11.0)
drupal/metatag (1.13.0 => 1.14.0)
drupal/publication_date (2.0.0-beta1 => 2.0.0-beta2)
drupal/search_api_autocomplete (1.3.0 => 1.4.0)
drupal/select2 (1.8.0 => 1.9.0)
drupal/tvi (1.0.0-beta2 => 1.0.0-rc2)
drupal/view_unpublished (1.0.0-rc3 => 1.0.0)
drupal/views_bulk_operations (3.8.0 => 3.9.0)
drupal/viewsreference (2.0.0-beta1 => 2.0.0-beta2)
drupal/webform (5.19.0 => 5.20.0)

- Check that foacl point, search, webform and views paragraph works.
